apps/ledger-live-mobile/src/families/ethereum/EthereumFeesStrategy.tsx

```diff
@@ -55,6 +57,34 @@ export default function EthereumFeesStrategy({
     [defaultStrategies, customStrategy],
   );
 
+  const disabledStrategies = useMemo(() => {
```

Comment:
If we edit a transaction we disable the fee strategy that are lower than the ones of the actual transaction